server/src/main/java/io/crate/replication/logical/MetadataTracker.java

```diff
@@ -95,12 +96,14 @@ private void stop() {
     }
 
     private void schedule() {
+        if (LOGGER.isDebugEnabled()) {
+            LOGGER.debug("Reschedule tracking metadata");
+        }
         cancellable = threadPool.schedule(
             this::run,
             pollDelay,
             ThreadPool.Names.LOGICAL_REPLICATION
         );
-        isActive = true;
```

Comment:
`isActive` should only be mutated at `start` and `stop`   as this could cause a conflict that the service continues to run when all subscriptions are removed right before the rescheduling.